aidial_adapter_bedrock/llm/converse/types.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
```

Comment:
Please leave a link to https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-runtime/client/converse.html as a reference

Reply:
Fixed